fixtures/beamz/cob30rgb-flatpar.json

```
@@ -0,0 +1,277 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "COB30RGB FlatPAR",
+  "shortName": "COB30RGB",
+  "categories": ["Color Changer", "Strobe"],
```

```suggestion
  "categories": ["Color Changer"],
```

It's an LED PAR -- it's not really what I'd call a strobe, even though (like virtually all LED fixtures) it has a strobe feature.